### PR TITLE
adds security/medical records to character setup 

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

anything put in will be injected into notes section

## Why It's Good For The Game

customization, roleplay, etc

## Changelog
:cl:
add: sec and medical records have been added to character setup.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
security record: is antag
medical record: is antag

**Comment by EmeraldSundisk** [comment]
This could be fun.

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1646,6 +1669,16 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(new_age)
 						age = max(min( round(text2num(new_age)), AGE_MAX),AGE_MIN)
 
+				if("security_records")
+					var/rec = stripped_multiline_input(usr, "Set your security record note section. This should be IC!", "Securitty Records", html_decode(security_records), MAX_FLAVOR_LEN, TRUE)
```
Comment (CameronWoof) [inline code/modules/client/preferences.dm]: Securitty > Security

File: code/modules/client/preferences.dm [hunk 2]
```diff
@@ -1646,6 +1669,16 @@ GLOBAL_LIST_EMPTY(preferences_datums)
...
+					if(!isnull(rec))
+						security_records = rec
+
+				if("medical_records")
+					var/rec = stripped_multiline_input(usr, "Set your medical record note section. This should be IC!", "Securitty Records", html_decode(medical_records), MAX_FLAVOR_LEN, TRUE)
```
Comment (CameronWoof) [inline code/modules/client/preferences.dm]: Ditto

**Review by Linzolle — APPROVED** [review]

**Pull request merged**